<-PYT:import moduletomock
<-PYT:import no_such_module
->RET:raise ImportError('No module named no_such_module')
<-PYT:moduletomock.no_such_method
->RET:raise AttributeError("'module' object doesn't REALLY have an attribute 'no_such_method', stupid!")
<-PYT:moduletomock.call_function('argument', faking=True)
->RET:raise TypeError('call_function() takes exactly 0 non-keyword arguments (1 given)')
<-PYT:import fakeurllib
<-PYT:fakeurllib.urlopen('foo://no.such.site')
->RET:raise Instance('URLError(IOError)', 'urlerror1')
<-PYT:urlerror1.args
->RET:('No such site, you wally!',)
<-PYT:fakeurllib.urlopen('foo://another.no.such.site')
->RET:raise Instance('URLError(IOError)', 'urlerror2')
<-PYT:urlerror2.__str__()
->RET:"<urlopen error (-2, 'Now you're really getting on my nerves!')>"
